src/app/contexts/RequestContext/index.jsx

```diff
@@ -22,6 +29,8 @@ export const RequestContextConsumer = ({ children }) => (
 RequestContextProvider.propTypes = {
   children: node.isRequired,
   platform: string.isRequired,
+  service: string.isRequired,
+  articleData: shape(articlePropTypes).isRequired,
```

Comment:
I don't think you should pass in all of `articleData` as a prop to the RequestContext. [I can see from your ATI PR that you're passing all of `articleData` to the ATI URL generator](https://github.com/bbc/simorgh/pull/1520/files#diff-3b8a6251365dbed373838ada59bd8682R20).
Whenever any part of the large JSON object `articleData` changes, [all of the RequestContext Consumers will re-render](https://reactjs.org/docs/context.html#contextprovider).  We do not want this to happen - we'd rather only pass down values that are strictly necessary. 

I think the only values that make sense to add to the Request Context are `isUK` & `env`, since we'd want a re-render of all of our components if we change environment or move within the UK to outside the UK or vice versa. 

All other values needed for the ATI URL - for example `optimoUrn`, `href`, `referrer`, `language`, `promoHeadline`, etc. - should be passed as standard props. The values can be extracted from the `articleData` [using the utilities that you've defined](https://github.com/bbc/simorgh/pull/1520/files#diff-c9f95ff3efb1cb76d379a304a81f7fa9R2).

Reply:
Also, I've noted that [in the ATI work, you're already passing in `articleData` as a prop to PageViewAnalytics](https://github.com/bbc/simorgh/pull/1520/files#diff-d5c925f82e0315a863ea6a24dd59cb60R47), so you really do not need to also pass it down via the `RequestContext`.